tests/test-addons.py

```diff
@@ -395,3 +396,14 @@ def test_backup_restore(self):
             os.remove('backupfile.tar.gz')
         check_call("/snap/bin/microk8s.dbctl --debug backup -o backupfile".split())
         check_call("/snap/bin/microk8s.dbctl --debug restore backupfile.tar.gz".split())
+
+    def test_falco(self):
+        """
+        Sets up and validates falco.
+        """
+        print("Enabling falco")
+        microk8s_enable("falco")
+        print("Validating falco")
+        validate_keda()
```

Comment:
There's some mistake on what is validating here.  I guess its should be `validate_falco`

Reply:
Sorry, got it, changed the same